src/sections/about.html

```diff
@@ -1,11 +1,14 @@
-<!DOCTYPE html>
-<html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>about</title>
-  </head>
-  <body class="font-outfit">
-    <h1 class="text-secondary">About</h1>
-  </body>
-</html>
+<div class="container mx-auto flex flex-col">
+  <p class="font-outfit px-8 py-14 md:text-2xl">
```

Comment:
Suggestion: I think instead of putting padding on the paragraph you can put it on the container for each content in it to have the same padding. Additional to that, it will prevent everyone in the team to add their own padding which , it will help you with the consistence in your design.